samples/excel/tables/format-table.yaml

```diff
@@ -0,0 +1,194 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Format table
+description: Format a table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#format-table').click(formatTable);
+
+    /** Create a new table with sample data */
+    async function createTable() {
+        try {
+            // Run a batch operation against the Excel object model
+            await Excel.run(async (context) => {
+                
+                // Force create the Playground sheet
+                await ExcelUtilities.forceCreateSheet(context, "Playground");
+
+                let sheet: Excel.Worksheet = context.workbook.worksheets.getItem("Playground");
+
+                // Call the helper to create the table
+                createTableHelper(sheet);
+                
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            });
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    /** Format the different components of the table*/
+    async function formatTable() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet: Excel.Worksheet;
+                    // Queue a command to get the sheet
+                    sheet = context.workbook.worksheets.getItem("Playground");
+
+                    // Call the helper to format the table
+                    formatTableHelper(sheet);
+
+                    // Run all of the queued up commands
+                    await context.sync();
+                });
+         }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    // Create the table
+    function createTableHelper(sheet: Excel.Worksheet) {
+
+            // Queue a command to add a table to the sheet
+            let expensesTable = sheet.tables.add('A1:D1', true);
+            expensesTable.name = "ExpensesTable";
+
+            // Queue a command to set the header row
+            expensesTable.getHeaderRowRange().values = [["DATE", "MERCHANT", "CATEGORY", "AMOUNT"]];
```

Comment:
Use title casing instead of all upper case. Upper case is harder to read and shouts.